dbt/loader.py

```diff
@@ -23,7 +24,17 @@ def load_all(cls, root_project, all_projects):
                     loader.load_all(root_project, all_projects, macros))
 
             to_return[subgraph] = subgraph_nodes
-
+        
+        model_fqns = []
+        for unique_id, node in to_return['nodes'].items():
```

Comment:
You can do this as a little one-liner with a [list comprehension](http://treyhunner.com/2015/12/python-list-comprehensions-now-in-color/). It will save ~2 lines of code, but the real benefit is that the logic is more self-evident. I think the same is true for `project_model_config_pqns` below